doc/README.md

```diff
@@ -387,7 +393,11 @@ you'll see the evolution of the page.
 While in Time Travel mode, you can drag on the page to pan around
 (like the
 [<img src="icons/arrows-alt.svg" width="18" alt="Pan Icon"> Pan Tool](#-pan-tool)),
-[<img src="icons/search-minus.svg" width="18" alt="Zoom Out Icon"> Zoom Out / <img src="icons/search-plus.svg" width="18" alt="Zoom In Icon"> Zoom In](#-zoom-out---zoom-in),
+pan using a mouse scroll-wheel or touchpad pan behavior,
+[<img src="icons/search-minus.svg" width="18" alt="Zoom Out Icon"> Zoom Out / <img src="icons/search-plus.svg" width="18" alt="Zoom In Icon"> Zoom In](#-zoom-out---zoom-in)
+using the buttons, using a mouse scroll-wheel while holding
+<kbd>Ctrl</kbd>, or using a touchpad zoom behavior,
+and
 [<img src="icons/download-svg.svg" width="18" alt="Download SVG Icon"> Download SVG](-download-svg) for the currently viewed version.
```

Comment:
There are three things we list as doable in Time Travel mode, each with lots of commas in them. Let's split them into a list, something like:
```
While in Time Travel mode, you can
* pan by dragging on the page ... or using a mouse scroll wheel ...
* zoom by ...
* download
```

Reply:
Very good idea. Hope you don't mind that I reordered the list, putting the undo/redo buttons at the front of the list, since those are most unique to this mode.